RecoBTag/SecondaryVertex/python/candidateNegativeCombinedSecondaryVertexComputer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.SecondaryVertex.candidateCombinedSecondaryVertexComputer_cfi import *
+
+candidateNegativeCombinedSecondaryVertexComputer = candidateCombinedSecondaryVertexComputer.clone()
+candidateNegativeCombinedSecondaryVertexComputer.vertexFlip = True
```

Comment:
it would be less verbose to put these overwritten parameters in the clone arguments